addOns/automation/CHANGELOG.md

```diff
@@ -4,8 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-### Added
-- Import Job profile (Issue 7078).
+### Added 
```

Comment:
The Import Job note should t be deleted

Reply:
accident :)

Reply:
Should I commit it back or :)... use some "git" manipulation (which I don't know how to do :))

Reply:
If you don’t mind I can do some git manipulation in a few hours and take care of it.

Reply:
I want to learn how to solve this mess :). Pretty sure I wont be able to make heavier contributions if I keep messing simple stuffs like these :).

Reply:
Since you're changes are relatively contained and straight forward I'd suggest resetting the branch to an up-to-date clean state then re-implementing all your changes in a single commit.
```
git fetch upstream
git reset --hard upstream/main
# Redo you changes here
commit -a -m "feat: automation: Add URL presence test" -m "Related to zaproxy/zaproxy#6461" --signoff
git push --force
```

Things to keep in mind for the future.
1. You can always use `git commit --amend --no-edit` after your first commit, your local commit history isn't relevant to the greater project (then you'll have to `git push --force`).
2. You should rebase instead of merging in order to update to latest upstream. You may still endup with conflicts but wont' have merge commits.

Reply:
To be clear when I say “redo changes” you can always come to the PR and copy/paste.

Reply:
Done